keps/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -141,11 +130,16 @@ proposing governance changes. However, as changes start impacting other SIGs or
 the larger developer community outside of a SIG, the KEP process should be used
 to coordinate and communicate.
 
-### KEP Template
+Enhancements that have major impacts on multiple SIGs should use the KEP process.
+A single SIG will own the KEP but it is expected that the set of approvers will span the impacted SIGs.
+The KEP process is the way that SIGs can negotiate and communicate changes that cross boundaries.
+
+KEPs will also be used to drive large changes that will cut across all parts of the project.
+These KEPs will be owned by SIG-architecture and should be seen as a way to communicate the most fundamental aspects of what Kubernetes is.
 
-The template for a KEP is precisely defined in the [template proposal][]
+### KEP Template
 
-[template proposal]: https://github.com/kubernetes/community/pull/1124
+The template for a KEP is precisely defined [here](0000-kep-template.md)
```

Comment:
probably better to not use a relative path here

Reply:
Relative paths are more useful for representing these KEPs someplace that isn't github.